lint/rule_enabled.go

```diff
@@ -77,29 +78,32 @@ func ruleIsEnabled(rule ProtoRule, d desc.Descriptor, l *dpb.SourceCodeInfo_Loca
 	}
 
 	// Some rules have a legacy name. We add it to the check list.
-	ruleName := string(rule.GetName())
-	names := []string{ruleName, aliasMap[ruleName]}
 
-	commentLines := []string{}
-	if l != nil {
-		commentLines = append(commentLines, strings.Split(l.GetLeadingComments(), "\n")...)
-	}
-	if f, ok := d.(*desc.FileDescriptor); ok {
-		commentLines = append(commentLines, strings.Split(fileHeader(f), "\n")...)
-	} else {
-		commentLines = append(commentLines, strings.Split(getLeadingComments(d), "\n")...)
-	}
-	disabledRules := []string{}
-	for _, commentLine := range commentLines {
-		r := extractDisabledRuleName(commentLine)
-		if r != "" {
-			disabledRules = append(disabledRules, r)
+	if !ignoreCommentDisables {
```

Comment:
I would move the entire contents of this block to a method like, `func ruleDisabledInline(...) bool` so that you don't have such a long block here.

Reply:
ack.